app/backend/src/ui/methods/projects/create.ts

```diff
@@ -9,9 +15,18 @@ import ProjectSchema from '../../../database/models/project';
  * @param messengerChannelUrl - project messenger channel url
  */
 export async function createProject(title: string, picture?: string, messengerChannelUrl?: string): Promise<Project> {
-  return await ProjectSchema.create({
+  const newProject = await ProjectSchema.create({
     title,
     picture,
     messengerChannelUrl,
   });
+
+  for (const label of PREDEFINED_STATUSES) {
```

Comment:
Please explain following code. Maybe it would better to move it to the separate method?

Reply:
moved to separate function